userspace/libsinsp/sinsp.cpp

```diff
@@ -1967,6 +1955,55 @@ void sinsp::get_capture_stats(scap_stats* stats) const
 	scap_get_stats(m_h, stats);
 }
 
+void sinsp::print_capture_stats(sinsp_logger::severity sev) const
+{
+	scap_stats stats;
+	get_capture_stats(&stats);
+
+	g_logger.format(sev,
+		"\nn_evts:%" PRIu64
+		"\nn_drops:%" PRIu64
+		"\nn_drops_buffer:%" PRIu64
+		"\nn_drops_buffer_clone_fork_enter:%" PRIu64
+		"\nn_drops_buffer_clone_fork_exit:%" PRIu64
+		"\nn_drops_buffer_execve_enter:%" PRIu64
+		"\nn_drops_buffer_execve_exit:%" PRIu64
```

Comment:
I've slightly changed the way in which we print the log. Now we don't have anymore a single line but we put a `\n` after each stat. This is an output example

```
08-21 11:32:31.914754 
n_evts:211937
n_drops:0
n_drops_buffer:0
n_drops_buffer_clone_fork_enter:0
n_drops_buffer_clone_fork_exit:0
n_drops_buffer_execve_enter:0
n_drops_buffer_execve_exit:0
n_drops_buffer_connect_enter:0
n_drops_buffer_connect_exit:0
n_drops_buffer_open_enter:0
n_drops_buffer_open_exit:0
n_drops_buffer_dir_file_enter:0
n_drops_buffer_dir_file_exit:0
n_drops_buffer_other_interest_enter:0
n_drops_buffer_other_interest_exit:0
n_drops_buffer_close_exit:0
n_drops_buffer_proc_exit:0
n_drops_scratch_map:0
n_drops_pf:0
n_drops_bug:0

08-21 11:32:31.914981 total threads in the table:1522, total fds in all threads:5319
```

So as you can see the output is quite small but if we should face issues in the future we could always revert it on one line, WDYT?

Reply:
It's used for debugging aka no problems at all would say